### PR TITLE
Replaced v2.wallabag with wallabag.it

### PR DESCRIPTION
**Pull request by nicosomb**
v2.wallabag.org will be closed in next days. We need to replace the URL in the android application.

### CODE REVIEW

File: app/src/main/java/fr/gaulupeau/apps/Poche/ui/preferences/ConnectionWizardActivity.java [hunk 1]
```diff
@@ -489,7 +489,7 @@ protected void gatherData() {
             username = usernameEditText.getText().toString();
             password = passwordEditText.getText().toString();
 
-            url = "https://v2.wallabag.org";
+            url = "https://www.wallabag.it";
```
Comment (di72nn) [inline app/src/main/java/fr/gaulupeau/apps/Poche/ui/preferences/ConnectionWizardActivity.java]: Shouldn't it be "https://app.wallabag.it"? I did not test it in the app, though.

Comment (nicosomb) [inline app/src/main/java/fr/gaulupeau/apps/Poche/ui/preferences/ConnectionWizardActivity.java]: Oops you are totally right. Too early for me ;-) !

**Comment by nicosomb** [comment]
Fixed.

**Comment by di72nn** [comment]
Well, it was necessary to fix the URL in the code, but all the string-resources can contain "wallabag.it" or whatever looks good.

**Comment by nicosomb** [comment]
Can we merge?

**Comment by nicosomb** [comment]
Nice. Do you know when can we publish the new version?

**Comment by Strubbl** [comment]
No. @tcitworld was always doing the new releases. Maybe he can answer.

**Pull request merged**